docs-source/content/userguide/managing-operators/using-the-operator/using-helm.md

```diff
@@ -125,6 +125,40 @@ imagePullSecrets:
 - name: "my-image-pull-secret"
 ```
 
+##### `nodeSelector`
+`nodeSelector` allows you to run the operator Pod on a node whose labels match the specified nodeSelector labels. You can use this optional feature if you want the operator Pod to run on a Node with particular labels. See [Assign Pods to Nodes](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector) in the Kubernetes documentation for more details. This is not required if the operator Pod can run on any Node.
+
+Example:
+```
+nodeSelector:
+  disktype: ssd
+```
+
+##### `nodeAffinity`
```

Comment:
`nodeAffinity` is conceptually similar to `nodeSelector`, it allows you to constrain... -> Allows you to constrain the operator Pod to be scheduled on a node with certain labels; conceptually similar to `nodeSelector`.
node  -> Node